views/js/pciCreator/dev/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -108,28 +108,20 @@ define([
          * @param {Number}  config.maxRecords - 0 = unlimited / 1 = no retry / x = x attempts
          * @param {Number}  config.maxRecordingTime - in seconds
          * @param {Boolean} config.isCompressed - set the recording format between compressed and uncompressed
+         * @param {Boolean} config.isLossless
          * @param {Number}  config.audioBitrate - number of bits per seconds for audio encoding
          * @param {Boolean} config.isStereo - switch the number of channels (1 vs 2) for uncompressed recording
          * @param {Boolean} config.useMediaStimulus - will display a media stimulus to the test taker
          * @param {Object}  config.media - media object (handled by the PCI media manager helper)
          * @param {Boolean} config.displayDownloadLink - for testing purposes only: allow to download the recorded file
+         * @param {String}  config.recordingFormat
+         * @param {Number}  config.compressionLevel
+         * @param {Number}  config.bps
```

Comment:
First comments are missing.

Second this is weird, we already have an `audioBitrate` parameter, so what is the difference with the `bps` parameter which means bits per second ? Unless it's a really different parameter but in that case the comments should mention it.
Also I think we should definitely prefix the options for the codec they refer to, having both `compressionLevel`, `audioBitrate`, etc. at the same level is very confusing.
If one option refers to flac, then `flacCompressionLevel` should be used, etc.

Reply:
`bps` means bits per sample, so it's not the sample like bitrate. I'll extend it with the comments and modify the variable names.